src/PrestaShopBundle/Resources/config/services/core/grid/doctrine_query_builder.yml

```diff
@@ -492,6 +492,11 @@ services:
       - '@=service("prestashop.adapter.shop.context").getContextListShopID()'
     public: true
 
+  prestashop.module.api.list.module_query_builder:
+    class: 'PrestaShop\Module\APIResources\List\ModuleQueryBuilder'
+    parent: 'prestashop.core.grid.abstract_query_builder'
+    autowire: true
+
```

Comment:
This shouldn't be in the core it depends on the module's code and should be in the module's services config (actually it's supposed to be in it already 🤔)